### ISSUE TITLE
Altered flow for new users

### ISSUE THREAD

**Issue by asimonw** [emfoundation/ce100-app#787]
When users log in, after creating a new password, they are guided through a different flow, asking them to populate their profile first.

To be determined (input from dwyl appreciated): should we do this alternate flow only on very first log in or as long as profile is not populated. UPDATE: this flow applies to all primary users who have not yet filled out a mission statement.

Copy for the message appearing at the top of the profile edit screen: 
"To get started, update your profile and let others know what you’re working on."

**Comment by asimonw** [comment]
Flow: Log in screen -> Edit Profile -> (on save) Profile
                                                         -> (on cancel) Walkthrough screen

**Comment by SimonLab** [comment]
We need to determine which fields need to be provided to determine the flow of the primary user. At the moment I think only the name of the organisation is required, which other fields will need to be required?
- [ ] name of the organisation
- [ ] logo
- [ ] Organisation's mission statement
- [ ] a minimum number of tags

@asimonw do you have an idea about these required fields?

**Comment by asimonw** [comment]
@SimonLab The name of the org is required, but will already be provided by the admin before the primary user logs in for the first time.

The only additional required field should be the mission statement. Only check that one to determine the primary user flow when logging in.

**Comment by SimonLab** [comment]
@asimonw do we want to display a message on the edit profile page (ex: "Please add a mission statement") to let know the user why she has been redirected? see https://github.com/emfoundation/ce100-app/pull/830#discussion_r117433430

**Comment by sohilpandya** [comment]
@asimonw as discussed on the call there are a few ways to tackle this issue:

- Add an extra block of text that would only appear when the user is redirected to the edit profile from login
- Add an extra block of text that would always appear if the mission statement is empty. (no matter how the user gets to the edit profile page)
- Update the copy so that it is universal and does not require adding to if a user has no mission statement. 

It would be great to hear back what your opinion is on the third option and whether that is viable.

**Comment by asimonw** [comment]
@sohilpandya I prefer your second option. As long as the mission statement is empty, keep the message. 
@SimonLab We'll indeed go for a block like that, but would like the colour to be blue, same as for information messages, like on the tag selection screen:
<img width="695" alt="screen shot 2017-05-19 at 16 31 11" src="https://cloud.githubusercontent.com/assets/15235804/26252521/b263dffc-3cb0-11e7-9fe9-e976aff9c4df.png">

On the other hand, I tried to test this by removing the mission statement for sa@ro.co, but it didn't work. Two problems:
* I was able to remove the mission statement, while it's supposed to be required now.
* I thought this would mean that next time I'd log in I'd see the profile edit screen, but I landed on the walkthrough.

**Comment by SimonLab** [comment]
I've tested again saving an empty mission statement and a correct error appears. I think when you've tested the texteara wasn't empty but had only white space, linked to https://github.com/emfoundation/ce100-app/issues/781#issuecomment-302728250

**Comment by asimonw** [comment]
@SimonLab You're right, apparently there was a second line of white space.Strange.
I'll test it with a new user now. In the meantime, the copy for the top message would be:
"To get started, update your profile and let others know what you’re working on."

**Comment by asimonw** [comment]
@SimonLab I tested it with a new users, and still a few issues:
* It doesn't work after activation, I needed to log out and in again to get the profile edit screen. In other words, this altered flow also needs to work coming from the password/activation screen.
* The mission statement already had some white space in it even before I ever touched it. I think the issue is related to the remaining issue in #781
